pygame/mixer.py

```diff
@@ -576,7 +576,7 @@ def _endsound_callback(channelnum):
 
     data = _channeldata[channelnum]
     # post sound ending event
-    if data.endevent != sdl.SDL_NOEVENT and sdl.SDL_WasInit(sdl.SDL_INIT_AUDIO):
+    if data.endevent != sdl.SDL_NOEVENT and sdl.SDL_WasInit(sdl.SDL_INIT_VIDEO):
```

Comment:
It's changed because checking for INIT_AUDIO is the wrong thing to check here. 

Calls to sdl.PushEvent need to be guarded by checks on INIT_VIDEO, because of SDL's assumptions, and that's what this should be checking. It's what's checked elsewhere and what  pygame checks in this situation. There's also no reason to check INIT_AUDIO - there shouldn't be any way to this part of the code without playing a sound, which implies that the audio has been initialised.